Makefile

```diff
@@ -11,14 +11,18 @@ EMBEDDED_OPERATOR_CHART_NAME = embedded-cluster-operator
 EMBEDDED_OPERATOR_CHART_VERSION = 0.36.5
 EMBEDDED_OPERATOR_UTILS_IMAGE = busybox:1.36.1
 EMBEDDED_CLUSTER_OPERATOR_IMAGE_OVERRIDE =
-OPENEBS_CHART_VERSION = 4.1.0
-OPENEBS_UTILS_VERSION = 4.1.0
+OPENEBS_CHART_VERSION = 4.0.1
+OPENEBS_IMAGE_VERSION = 4.0.0@sha256:9e45763b9d067bfef4b367b7b29a4d79d477897a8c7f9a16f4839a3a7e4b31a8 
```

Comment:
This caught my eye, I looked into the helm chart and it seems to me that this should be `4.1.0` instead of `4.0.0`.